Readme_ES.md

```diff
@@ -0,0 +1,307 @@
+# Paquete GT para moderadores de New Horizons
```

Comment:
Let's do some literal translations here: "GT Package for moderators of New Horizons".